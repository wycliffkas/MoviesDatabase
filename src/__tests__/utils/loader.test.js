import Loader from "../../utils/Loader";
import { shallow } from "enzyme";
import React from "react";

describe("Loader Test", () => {
  const wrapper = shallow(<Loader />);
  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
