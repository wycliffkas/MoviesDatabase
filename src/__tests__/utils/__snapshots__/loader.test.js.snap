// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Test should render correctly 1`] = `ShallowWrapper {}`;
